backend/backend/pay_tax.py

```diff
@@ -0,0 +1,15 @@
+""" Module providing functionality for players to pay tax. """
+
+from backend.player import Player
+
+
+def pay_tax(player_id, tax_payable):
+    """Deduct tax payable from player's balance.
+
+    Arguments:
+        player_id: An int representing the player_id.
+        tax_payable: An int representing the money to deduct from player.
+
+    """
+    player = Player(player_id)
+    player.balance -= tax_payable
```

Comment:
I think this bit should be in a `with` block, as the player object is being updated.

Reply:
Yes it should. Thanks for pointing that out.